File: exercises/template.py
import sys

{{SOLUTION}}

inputS = '{{INPUT}}'
expectedS = '{{OUTPUT}}'
expected  = eval(expectedS)

try:
    result  = solution({{INPUT}})
    resultS = repr(result)
    
    if resultS != expectedS:
        #raise "solution(%s) => %s     Expected: %s" % (inputS, resultS, expectedS)
        sys.stderr.write("solution(%s) => %s     Expected: %s\n" % (inputS, resultS, expectedS) )
    else:
        print "solution(%s) => %s     OK!" % (inputS, resultS)
except BaseException as err:
    sys.stderr.writeln(err)

File: runtimes/js/template.js
var _assert_ = require('assert');

module._load = function(request/*, parent, isMain*/) {
    console.error('attempted to require module "' + request + '"!');
};

{{SOLUTION}}

var inputS    = "{{INPUT}}";
var expectedS = "{{OUTPUT}}";
var expected  =  {{OUTPUT}};

try {
    var result  = solution({{INPUT}});

    _assert_.notStrictEqual(result, undefined, 'solution has not returned any value!');

    var resultS = JSON.stringify(result).replace(/"/g, "'");

    _assert_.deepEqual(result, expected);
    
    console.log( ['solution(', inputS, ') => ', resultS, '     OK!'].join('') );
} catch (ex) {
    console.error(ex);
}
